src/js/actions/addScale.js

```diff
@@ -9,7 +9,7 @@ function addScale(data) {
   }, data);
 
   var action = {
-    id: props._id,
+    id: 'scale_' + props._id,
```

Comment:
I was having issues retrieving from the store with a key as an integer. For marks all the ids are always strings even when they are numbers.  I added that to test this theory and make things work so I could get the PR in asap.